internal/testrunner/runners/system/servicedeployer/_static/terraform_deployer_run.sh

```diff
@@ -20,10 +20,12 @@ trap cleanup EXIT INT TERM
 
 terraform init
 terraform plan
-terraform apply -auto-approve && touch /tmp/tf-applied
+terraform apply -auto-approve
 
 terraform output -json > /output/tfOutputValues.json
 
+touch /tmp/tf-applied
```

Comment:
Should we document that this should be last statement in the script to avoid future issues?

```suggestion
touch /tmp/tf-applied # This is an indicator that the service is UP and so should be the last statement in the script
```